pkg/controller/plan/controller.go

```diff
@@ -198,6 +198,12 @@ func (r Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (r
 		return
 	}
 
+	// Don't reconcile succeeded plans unless they need archiving.
+	if plan.Status.HasCondition(Succeeded) && !plan.Spec.Archived {
+		r.Log.V(1).Info("Skipping reconcile of succeeded plan.", "plan", plan.Name)
+		return reconcile.Result{RequeueAfter: 0}, nil
```

Comment:
please use the `result.RequeueAfter = 0 ` and `return`
Just keeping it consistent across the code